data_visualization_demo.ipynb

```diff
@@ -0,0 +1,3079 @@
+      "Requirement already satisfied: locket in /opt/conda/lib/python3.9/site-packages (from partd>=0.3.10->dask[complete]->xarray[complete]) (0.2.0)\n",
+      "Requirement already satisfied: charset-normalizer~=2.0.0 in /opt/conda/lib/python3.9/site-packages (from requests>=2.19.0->pooch->xarray[complete]) (2.0.12)\n",
+      "Requirement already satisfied: urllib3<1.27,>=1.21.1 in /opt/conda/lib/python3.9/site-packages (from requests>=2.19.0->pooch->xarray[complete]) (1.26.8)\n",
+      "Requirement already satisfied: idna<4,>=2.5 in /opt/conda/lib/python3.9/site-packages (from requests>=2.19.0->pooch->xarray[complete]) (3.3)\n",
+      "Requirement already satisfied: soupsieve>1.2 in /opt/conda/lib/python3.9/site-packages (from beautifulsoup4->pydap->xarray[complete]) (2.3.1)\n",
+      "Requirement already satisfied: llvmlite<0.39,>=0.38.0rc1 in /opt/conda/lib/python3.9/site-packages (from numba->numbagg->xarray[complete]) (0.38.0)\n",
+      "Requirement already satisfied: heapdict in /opt/conda/lib/python3.9/site-packages (from zict>=0.1.3->distributed==2022.02.1->dask[complete]->xarray[complete]) (1.0.1)\n",
+      "Requirement already satisfied: pycparser in /opt/conda/lib/python3.9/site-packages (from cffi->eccodes>=0.9.8->cfgrib->xarray[complete]) (2.21)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install zarr\n",
+    "!pip install earthpy\n",
+    "!pip install shapely\n",
+    "!pip install cartopy\n",
+    "!pip install geopandas\n",
+    "!pip install rioxarray\n",
+    "!pip install xarray[complete]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "d80051d6-058b-4e6e-99ad-4011d06562cf",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/opt/conda/lib/python3.9/site-packages/xarray/backends/cfgrib_.py:27: UserWarning: Failed to load cfgrib - most likely there is a problem accessing the ecCodes library. Try `import cfgrib` to get the full error message\n",
+      "  warnings.warn(\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import s3fs\n",
+    "import glob\n",
+    "import zarr\n",
+    "import boto3\n",
+    "import fsspec\n",
+    "import pathlib\n",
+    "import dask.array\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import xarray as xr\n",
+    "import geopandas as gpd\n",
+    "import rioxarray as rxr\n",
+    "\n",
+    "import rasterio\n",
+    "from rasterio.crs import CRS\n",
+    "from rasterio.transform import Affine\n",
+    "from rasterio.plot import show, plotting_extent\n",
+    "\n",
+    "import cartopy\n",
+    "import cartopy.crs as ccrs\n",
+    "\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from shapely.geometry import box, mapping, shape, Point, LineString, MultiPolygon, LinearRing, Polygon\n",
+    "import cartopy.feature as cfeature"
```

Comment:
Helpful for ignoring warnings:
import warnings
warnings.filterwarnings('ignore')